src/Command/GenerateDashCommand.php

```diff
@@ -0,0 +1,241 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\Console\Command\GenerateDocCommand.
+ */
+
+namespace Drupal\Console\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Filesystem\Exception\IOExceptionInterface;
+
+class GenerateDashCommand extends ContainerAwareCommand
+{
+
+    /**
+     * @constant Contents of the plist file required by the docset format.
+     */
+    const PLIST = <<<PLIST
```

Comment:
Maybe we can put this ```const``` into a twig file like the html code? cc @FatherShawn @enzolutions